### DISCUSSION TITLE
Unable to open zarr files

### DISCUSSION THREAD

**Discussion by sergibech** [pydata/xarray#5619] [Q&A] [answered]
Hi! I'm trying to open a zarr file and I have installed all the dependencies following the xarray documentation. However, I get the next error all the time. It seems a cftime error but I also have installed the cftime dependency.

![imagen](https://user-images.githubusercontent.com/29817603/126125143-9b6bb46b-3f98-4524-8c4c-1fc50b01c46d.png)

![imagen](https://user-images.githubusercontent.com/29817603/126124702-a20b490e-f0b8-4a75-a11f-d28055d17e4f.png)

If anyone knows how to solve it I would appreciate it a lot. Also I try with different zarr files and I always get the same error.

Thanks!

**Comment by Illviljan** [comment]
Try updating `dask` to the latest version.

**Comment by sergibech** [comment, marked as answer]
I solved the issue. It was a problem of jupyter notebook not loading the correct environment.

Thanks!
